.github/workflows/README.md

```
@@ -1,394 +1,188 @@
-- Push to `main`
-- Pull requests
-
-**Jobs:**
-1. `rustfmt` - Check Rust formatting via `make fmt:check:rust`
-2. `clippy` - Run Clippy linter via `make clippy` on all platforms
-3. `python` - Run Python lint and format checks via `make lint:python` and `make fmt:check:python`
-4. `node` - Run Node lint and format checks via `make lint:node` and `make fmt:check:node`
-5. `c` - Run C SDK lint and format checks via `make lint:c` and `make fmt:check:c`
-
-### `codeql.yml`
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)